hdk-rust/tests/integration_test.rs

```diff
@@ -665,7 +665,7 @@ fn can_send_and_receive() {
     assert!(result.is_ok(), "result = {:?}", result);
     let agent_id = result.unwrap().to_string();
 
-    let (mut hc2, _) = start_holochain_instance("can_remove_modified_entry", "bob");
+    let (mut hc2, _) = start_holochain_instance("can_send_and_receive", "bob");
```

Comment:
Wasn't sure about this one, seemed like the test shouldn't even pass without this change, can someone explain if it was actually supposed to be the other way?

Reply:
I've added this parameter that gets set as the DNA's UUID in order to not have tests interfere with each other since most of our tests create the same boilerplate DNA that then happens to have the same DNA hash which results in instances of different tests communicating with each other through the mock network.. 

So yes, this change is good. But the problem only appears if tests creating the same DNA hash run simultaneously.